e2e/test/scenarios/visualizations/smartscalar-trend.cy.spec.js

```diff
@@ -29,7 +29,5 @@ describe("scenarios > visualizations > scalar", () => {
     cy.log("Bug: showing blank visualization");
 
     cy.get(".ScalarValue").contains("100");
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Nothing to compare for the previous month.");
```

Comment:
I removed this check because it’s only necessary to confirm that this visualization displays a value, not what the comparison is.  See #13710.

_Extra context— Our new frontend `computeTrend` function skips over null gaps until finding a valid month, and changes this message.  I opted not to correct this message because the rows are not sorted by date here, which causes the backend trend calculation to trip up since the previous row is not exactly one month adjacent to the source row.  The frontend calculation now displays the previous row regardless of what it is._